public/Lab-Files/Lesson03/Challenges/wild-animal.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Wild Animal</title>
+    <style>
+        .fox {
+            width: 500px;
+            display: block;
+            margin: auto;
+            border: 5px solid #ff9dff;
+        }
+        
+        h1 {
+            font-family: Arial Black;
+            font-size: 50px;
+            text-align: center;
+            border: 5px solid #00bfff;
+            margin-left: 125px;
+            margin-right: 125px;
+        }
+        
+        h2 {
+            font-family: cursive;
+            text-align: center;
+            font-size: 40px
+        }
+        
+        .cold {
+            color: #00bfff;
+        }
+        
+        p {
+            font-family: Arial;
+            margin: 50px;
+        }
+        
+        .citation {
+            font-style: oblique;
+            text-align: center;
+        }
+        
+        .brown {
+            color: brown;
+        }
+        
+        body {
+            background-image: url(snow.jpg);
+        }
+        
+    </style>
+</head>
+<body>
+    <main>
+        <section>
+        <h1 class="coolest">The Coolest Creature</h1>
+        <h2><em>The Artic Fox</em></h2>
+        <img src="ArcticFox.jpg" alt="Artic Fox Leaves its Den" class="fox">
+        <p>The Arctic fox (<em>Vulpes lagopus</em>), also known as the white fox, polar fox, or snow fox, is a small fox native to the Arctic regions of the Northern Hemisphere and common throughout the Arctic tundra biome. It is well adapted to living in <span class="cold">cold environments</span>. It has a deep, thick fur which is <span class="brown">brown</span> in summer and white in winter. Its body length ranges from 46 to 68 cm (18 to 27 in), with a generally rounded body shape to minimize the escape of body heat.
+The Arctic fox preys on any small creatures such as: lemmings, voles, ringed seal pups, fish, waterfowl, and seabirds. It also eats carrion, berries, seaweed, insects, and other small invertebrates. Arctic foxes form monogamous pairs during the breeding season and they stay together to raise their young in complex underground dens. Occasionally, other family members may assist in raising their young.</p>
+        </section>
+        <footer class="citation">Text from Wikipedia</footer>
+    </main>
+</body>
+</html>
```

Comment:
This looks great! Awesome job! I like that you keep your code organized, it's very easy to read and follow.